js/components/advanced-options.tsx

```diff
@@ -31,7 +31,7 @@ interface IState {}
 @observer
 export class AdvancedOptions extends BaseComponent<IProps, IState> {
   changeInteraction: (value: any) => void;
-  changeTimestep: (value: any) => void;
+  changeTargetModelStepsPerSecond: (value: any) => void;
```

Comment:
```suggestion
  changeTargetModelStepsPerSecond: (value: number) => void;
```
I would think this could be typed at this point.